apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -292,18 +297,24 @@ const DepartureTimes = ({
   );
 
   const schedules = schedulesByHeadsign(schedulesForDirection);
-
   return (
     <>
       {Object.entries(schedules).map(([headsign, schs]) => {
-        const formattedTimes = toDisplayTime(
-          schs,
-          predictionsByHeadsign[headsign]
-            ? predictionsByHeadsign[headsign]
-            : [],
-          overrideDate
+        const preds = predictionsByHeadsign[headsign]
+          ? predictionsByHeadsign[headsign]
+          : [];
+        const formattedTimes = toDisplayTime(schs, preds, overrideDate);
+        return (
+          <div
+            className="departure-row-click-test"
```

Comment:
In the future we should query by [test ids](https://testing-library.com/docs/queries/bytestid) instead of class names.  This should only be used if all other query methods are not working.